### DISCUSSION TITLE
How to package tinymce.min.js and plugin.min.js into one file

### DISCUSSION THREAD

**Discussion by limeng32** [tinymce/tinymce#8779] [General]
It's not a bug, just a suggestion, or a feature that already exists but I don't know how to use;

**Comment by lostkeys** [comment]
Hi @limeng32, have a look at the [documentation about bundling TinyMCE](https://www.tiny.cloud/docs/tinymce/6/introduction-to-bundling-tinymce/)

— Fredrik, Product Manager
